ps_emailalerts.php

```diff
@@ -115,7 +115,13 @@ public function install($delete_params = true)
             return false;
         }
 
-        if ($delete_params && $this->uninstallPrestaShop16Module()) {
+        // handle PS16
+        $moduleInstalledOn16 = (Module::isInstalled(self::PS_16_EQUIVALENT_MODULE));
+        if ($moduleInstalledOn16 && (!$this->uninstallPrestaShop16Module())) {
```

Comment:
the `uninstallPrestaShop16Module` always returns true :thinking:

Reply:
Indeed. Maybe I should do `return $parentUninstallClosure();` to obtain the result of the uninstall process ?

Reply:
maybe :sweat_smile:

Reply:
Fixed in https://github.com/PrestaShop/ps_emailalerts/pull/64/commits/12e2824fcf825f02f593f1929f27b8333767c309